packages/design-system/src/components/ai-command-bar/command-bar.tsx

```diff
@@ -0,0 +1,103 @@
+import * as Collapsible from "@radix-ui/react-collapsible";
+import * as Separator from "@radix-ui/react-separator";
+import { styled, theme } from "../../stitches.config";
+import { Grid } from "../grid";
+import { Button } from "../button";
+import { type ReactNode } from "react";
+
+type Props = {
+  children: React.ReactNode;
+  content?: React.ReactNode;
+  open: boolean;
+  onOpenChange: (open: boolean) => void;
+};
+
+const borderRadius = 22;
+
+const GridContainer = styled(Grid, {
+  gridTemplateColumns: `max-content 1fr`,
+  gridAutoColumns: "max-content",
+  gridAutoFlow: "column",
+  padding: theme.spacing[5],
+  backgroundColor: "#1D1D1D",
+  borderRadius,
+  variants: {
+    open: {
+      true: {
+        borderTopLeftRadius: 0,
+        borderTopRightRadius: 0,
+        borderTop: "1px solid #323232",
+      },
+    },
+  },
+});
+
+const CollapsibleRoot = styled(Collapsible.Root, {
+  color: theme.colors.foregroundContrastMain,
+  backgroundColor: "#1D1D1D",
+  borderRadius,
+  border: "1px solid #323232",
+  boxShadow:
+    "0px 5px 17px 0px rgba(0, 0, 0, 0.30), 0px 2px 7px 0px rgba(0, 0, 0, 0.10)",
```

Comment:
add a @todo please everywhere where we need to update tokens and work left overs